server/src/com/cloud/network/element/VirtualRouterElement.java

```diff
@@ -227,7 +227,7 @@ public boolean implement(final Network network, final NetworkOffering offering,
         final List<DomainRouterVO> routers = routerDeploymentDefinition.deployVirtualRouter();
 
         int routerCounts = 1;
```

Comment:
Instead of `routerCounts`, what about `expectedNumberRouter` or something similar. This is the number of VRs you expect to have in a network, right?

Reply:
@rafaelweingartner good idea, maybe for a next/different refactoring PR.

Reply:
Fixed.